src/mixins/trackHelper.js

```diff
@@ -31,6 +31,10 @@ export var getTrackCSS = function(spec) {
     msTransform: 'translateX(' + spec.left + 'px)'
   };
 
+  if (!-[1,]) {
```

Comment:
Please hold on for a sec, the current array in IE8 still has problem, I will need to push a fix.